src/js/utils/ui.js

```diff
@@ -378,7 +378,12 @@ function removeHandlers(ui, triggerName) {
     const listenerOptions = options[triggerName];
     if (listeners) {
         Object.keys(listeners).forEach(domEventName => {
-            element.removeEventListener(domEventName, listeners[domEventName], listenerOptions);
+            const useCapture = listenerOptions[domEventName];
```

Comment:
I am not sure if we need to check for boolean value.
`the only option removeEventListener() checks is the capture/useCapture flag. Its value must match for removeEventListener() to match, but the other values don't.`
Doesn't this mean that we can just pass in the same options as we did for `addEventListener`, and the options other than capture(boolean)/useCapture(boolean in an object) are ignored?

Reply:
See my comment below https://github.com/jwplayer/jwplayer/pull/3226#discussion_r241848296

Passing in any object will cause the bug in IE 11 that these changes fix.